src/style.css

```diff
@@ -0,0 +1,77 @@
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  font-family: Arial, sans-serif;
+  color: rgb(255, 255, 255);
+  font-size: 100px;
```

Comment:
it's alright. but generally, you want to set it for the HTML tag and only change it for selectors that use a different font size or color. font-family, for the most part, stays the same across the website